lib/pure/strutils.nim

```diff
@@ -325,7 +394,8 @@ proc toOctal*(c: char): string {.noSideEffect, rtl, extern: "nsuToOctal".} =
     result[i] = chr(val mod 8 + ord('0'))
     val = val div 8
 
-iterator split*(s: string, seps: set[char] = Whitespace, maxsplit: int = -1): string =
+iterator split*(s: string, seps: set[char] = Whitespace, maxsplit: int = -1,
+                keepSeparators: bool = false): string =
```

Comment:
Maybe you're right. I was following the idea stated [here](https://github.com/nim-lang/Nim/issues/4251), but it's probably not that useful in most cases, since you already know what the separator is. I'll do away with these, then.

Interesting tidbit of information. I didn't know that 2 yield statements would insert the loop body again. In situations where you want to use 2 yields in a loop, should you not use an iterator?